app/services/zendesk_service.rb

```diff
@@ -33,6 +33,41 @@ def self.destroy_tickets!(ids)
   end
 
   def self.ticket_template(trn_request)
+    if trn_request.from_get_an_identity?
+      identity_ticket_template(trn_request)
+    else
+      find_a_lost_trn_ticket_template(trn_request)
+    end
+  end
+
+  def self.identity_ticket_template(trn_request)
```

Comment:
Not necessarily for this PR but there's some template duplication here we could look to golf down in future.
It seems like the subject, custom_fields value and comment value differ, and in the case of the comment value we're only appending another line. 
Perhaps the templating could be DRYer with `Array#join("\n")`.
Not a blocker as this follows the previous implementation.

Reply:
Yeh, I think this second instance duplication is easier to understand than something more complex, but if there's a 3rd instance then we'll definitely revisit this